tests/parm/nems.configure.cpld.IN

```diff
@@ -104,6 +104,7 @@ runSeq::
      MED med_phases_post_ice
      WAV -> MED :remapMethod=redist
      MED med_phases_post_wav
+     MED med_phases_ocnalb_run
```

Comment:
Are the changes backward compatible? Can you create an global-workflow issue to let people know the nems.configure changes for the coupled model?

Reply:
Yes, I've created issue https://github.com/NOAA-EMC/global-workflow/issues/1811